eno-core/src/main/java/fr/insee/eno/core/converter/LunaticConverter.java

```diff
@@ -74,10 +74,16 @@ else if (enoObject instanceof TableCell)
 
     private static Object instantiateFrom(SingleResponseQuestion enoQuestion) {
         if (enoQuestion instanceof TextQuestion textQuestion) {
-            if (textQuestion.getMaxLength().intValue() < Constant.LUNATIC_SMALL_TEXT_LIMIT)
-                return new Input();
-            else
-                return new Textarea();
+            if (textQuestion.getMaxLength().intValue() < Constant.LUNATIC_SMALL_TEXT_LIMIT) {
```

Comment:
"else" is devil =) cognitive complexity could be significatively reduced here with early returns

Reply:
True. I opened an issue for this https://github.com/InseeFr/Eno/issues/583